Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -0,0 +1,107 @@
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+
+__Improved Narrator support__
+- Narrator will now announce the value of the `P:System.Windows.Forms.Keys.ShortcutKeys` property when announcing the text of a `T:System.Windows.Forms.ToolStripMenuItem`. 
+- Narrator will now indicate when a `T:System.Windows.Forms.ToolStripMenuItem` has its `P:System.Windows.Forms.ToolStripMenuItem.Enabled` property to _false_.
+- Narrator will now give feedback on the state of a check box when the `P:System.Windows.Forms.ListView.CheckBoxes` property is set to _true_.
+- Narrator Scan Mode focus order is now consistent with the visual order of the controls on the ClickOnce download dialog window.
+
+__Improved DataGridView Accessibility support__
+- Rows in a `T:Systm.Windows.Forms.DataGridView` can now be sorted using the keyboard. Now a user can use the F3 key in order to sort by the current column.
+- When the `P:System.Windows.Forms.DataGridView.SelectionMode` property of a `T:System.Windows.Forms.DataGridView` is set to <xref:System.Windows.Forms.DataGridViewSelectionMode.FullRowSelect?displayProperty=nameWithType> the column header will change color to indicate the current column as the user tabs through the cells in the current row.
+- The `P:System.Windows.Forms.DataGridViewCell.DataGridViewCellAccessibleObject.Parent` property of a `T:System.Windows.Forms.DataGridViewCell.DataGridViewCellAccessibleObject` now returns the correct parent control.
+
+__Improved Visual cues__
+- `T:System.Windows.Forms.RadioButton` and `T:System.Windows.Forms.CheckBox` controls with an empty Text property will now display a focus indicator when they recieve focus.
+
+__Improved Property Grid Support__
+- The `T:System.Windows.Forms.PropertyGrid` control child elements will now return a _true_ for the [IsEnabled](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationelement.isenabledproperty) property only when a PropertyGrid element is enabled.
+- The `T:System.Windows.Forms.PropertyGrid` control child elements will now return a _false_ for the [IsReadOnly](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.valuepattern.isreadonlyproperty) property only when a PropertyGrid element can be changed by the user.
+
+For an overview of UI automation, see the [UI Automation Overview](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-overview).
+
+
+__Improved keyboard navigation__
+- `T:System.Windows.Forms.ToolStripButton` now allows focus when contained within a `T:System.Windows.Forms.ToolStripPanel` which has the `P:System.Windows.Forms.TabStop` property set to _true_

```

Comment:
which --> that
TabStop: \<xref:System.Windows.Forms.ToolStripPanel.TabStop>. The current documentation says that the TabStop property "is not relevant to this class." Is it correct if changed to, "Determines whether individual ToolStripButton controls contained within the ToolStripPanel can receive the focus."?
fence true

Reply:
Yes, that is now correct with this fix.